packages/sitecore-jss-react/src/components/PlaceholderMetadata.test.tsx

```diff
@@ -0,0 +1,109 @@
+﻿import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import { PlaceholderMetadata } from './PlaceholderMetadata';
+
+/**
+ * Normalize HTML output for comparison
+ * @param str
+ */
+function normalizeWhitespace(str) {
+  return str.replace(/\s+/g, ' ').trim();
+}
+
+describe('PlaceholderWithMetadata', () => {
+  it('renders a single component without placeholders', () => {
+    const component = {
+      uid: '123',
+      componentName: 'RichText',
+    };
+
+    const wrapper = shallow(<PlaceholderMetadata rendering={component} />);
+
+    const expectedOutput =
+      '<code type="text/sitecore" chrometype="rendering" class="scpm" kind="open" id="123"></code><RichText></RichText><code type="text/sitecore" chrometype="rendering" class="scpm" kind="close" id="123"></code>';
```

Comment:
Would it be possible to make this more readable? Kind of like this:
https://github.com/Sitecore/jss/pull/1773/files#diff-77b3f451fd7b04ab45e786f4f22abe108550edb30a868b1949f8a814ac9d18d5R106-R114 ?

Reply:
Also, is it expected here for RichText component to not be rendered as HTML? Should this be full SSR instead?

Reply:
It was just being rendered as a string. Now it is html.